bridge/engine/AudioForwarderReceiveJob.cpp

```diff
@@ -147,6 +158,7 @@ void AudioForwarderReceiveJob::run()
         }
 
         _activeMediaList.onNewAudioLevel(_sender->getEndpointIdHash(), audioLevel);
+        _activeMediaList.onNewPtt(_sender->getEndpointIdHash(), is_ptt);
```

Comment:
Do not make the call if we do not have header.

Reply:
Fixed.